apm-server.yml

```diff
@@ -114,6 +114,26 @@ apm-server:
     # Set to false to disable the metrics endpoint
     #enabled: true
 
+  # A pipeline is a definition of processors applied to documents when writing them to Elasticsearch.
+  # Using pipelines involves two steps:
+  # (1) registering a pipeline
+  # (2) applying a pipeline during data ingestion (see `output.elasticsearch.pipelines`)
+  #
+  # You can manually register pipelines, or use this configuration option to ensure
+  # pipelines are loaded and registered at the configured Elasticsearch instances.
+  # Automatic pipeline registration requires 
+  # * `output.elasticsearch` to be enabled and configured.
```

Comment:
thanks for updating the docs, much better now!
so there are 2 dependent settings, in `output.elasticsearch.pipelines` you refer to `apm-server.register.ingest.pipeline`, but not the other way around.

would be great to add another line like  
* `output.elasticsearch.pipelines` contains `"apm_user_agent"` or similar

Reply:
I refer to them both ways, see 
* line 120: (2) applying a pipeline during data ingestion (see `output.elasticsearch.pipelines`)
* line 400: # (1) ensure pipelines are registered in Elasticsearch, see `apm-server.register.ingest.pipeline`

Reply:
missed L120, sorry